.github/workflows/docker-build-and-push.yml

```
@@ -0,0 +1,74 @@
+name: Build and Push Docker Image
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+  actions: read
+  security-events: write
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
+jobs:
+  docker-build:
+    runs-on: depot-ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Depot
+        uses: depot/setup-action@v1
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GHCR_USERNAME }}
+          password: ${{ secrets.GHCR_TOKEN }}
```

can you change this to:

```suggestion
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```

I'd like to try this so we can stop futzing with tokens when we are publishing a package with the same name as the repo.

---

Fixed